docs/configuration/config-file-reference.md

```diff
@@ -149,6 +165,31 @@ The `server_config` configures the HTTP and gRPC server of the launched service(
 # CLI flag: -server.grpc-max-concurrent-streams
 [grpc_server_max_concurrent_streams: <int> | default = 100]
 
+# The duration after which an idle connection should be closed. Default:
+# infinity
+# CLI flag: -server.grpc.keepalive.max-connection-idle
+[grpc_server_max_connection_idle: <duration> | default = 2562047h47m16.854775807s]
```

Comment:
`2562047h47m16.854775807s` is the default but it looks so strange here haha. Don't think there is much we can do about it.

Reply:
Agree. We should have addressed it in `weaveworks/common`, like defaulting it to `0` in the config and then internally replacing it with the max value. We could keep it for a future improvement.